test/Generator/PropertyGeneratorTest.php

```diff
@@ -330,4 +380,20 @@ public function testFromReflectionOmitsTypeHintInTypedProperty(): void
 
         self::assertSame('    private $typedProperty;', $code);
     }
+
+    public function testFromReflectionReadonlyProperty(): void
+    {
+        if (false === strpos((string)phpversion(), '8.1')) {
```

Comment:
This can be `/** @requires php>=8.1 */` at docblock level :+1:

Reply:
Syntax is `/** @requires PHP >= 8.1 */`